### PR TITLE
Better solution for scenario 1

### PR DESCRIPTION
**Pull request by ArmandoLacerda**
The third paragraph if scenario 1 is confusing and technically deficient. One cannot do ETL using the SQL endpoint in a lakehouse, which seems to be the intent.  Here is my suggestion for a cleaner, more focused solution Susan would adopt.

# Thank you for contributing to Microsoft Fabric documentation

## Fill out these items before submitting your pull request:

If you are working internally at Microsoft:
**Provide a link to an Azure DevOps Boards work item that tracks this feature/update.**

>
>

**Who is your primary Skilling team contact?** \@mention them individually tag them and let them review the PR before signing off.

>
>

## For internal Microsoft contributors, check off these quality control items as you go

- [ ] 1. **Check the Acrolinx report:** Make sure your Acrolinx Total score is **above 80 minimum** (higher is better) and with no spelling issues. Acrolinx ensures we are providing consistent terminology and using an appropriate voice and tone, and helps with localization.

- [ ] 2. **Successful build with no warnings or suggestions**: Review the build status to make sure **all files are green** (Succeeded).

- [ ] 3. **Preview the pages:**: Click each **Preview URL** link to view the rendered HTML pages on the review.learn.microsoft.com site to check the formatting and alignment of the page. Scan the page for overall formatting, and look at the parts you edited in detail.

- [ ] 4. **Check the Table of Contents:** If you are adding a new markdown file, make sure it is linked from the table of contents.

- [ ] 5. **#sign-off to request PR review and merge**: Once the pull request is finalized and ready to be merged, indicate so by typing `#sign-off` in a new comment in the Pull Request. If you need to cancel that sign-off, type `#hold-off` instead. *Signing off means the document can be published at any time.* Note, this is a formatting and standards review, not a technical review.

## Merge and publish
- After you `#sign-off`, there is a separate PR Review team that will review the PR and describe any necessary feedback before merging. 
- The review team will use the comments section in the PR to provide feedback if changes are needed. Address any blocking issues and sign off again to request another review.
- Once all feedback is resolved, you can `#sign-off` again. The PR Review team reviews and merges the pull request into the specified branch (usually the main branch or a release- branch).
- From the main branch, the change is merged into the live branch several times a day to publish it to the public learn.microsoft.com site.

### CODE REVIEW

**Comment by Court72** [comment]
@bradleyschacht

Can you review the proposed changes? 
	
**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 
	
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by bradleyschacht** [comment]
The text in the rewritten paragraph convey many of the correct concepts, but do not make sense as written. 

**Current text:**
Susan creates a new data warehouse and access it as it is a SQL server database. Using the Fabric portal, she creates references to other lakehouses and data warehouses using cross-database query technics. Susan now can write T-SQL queries that reference Delta Tables in the lakehouse. She can query data using T-SQL three-part names.

**Proposed text:**
Susan creates a new data warehouse and interacts with it using T-SQL just like her other SQL server databases. Most of the existing T-SQL code she has written to build her warehouse on SQL Server will work on the Fabric data warehouse making the transition easy. If she chooeses to, she can even use the same tools that work with her other databases, like SQL Server Management Studio. Using the SQL editor in the Fabric portal, Susan and other team members write analytic queries that reference other data warehouses and Delta tables in lakehouses simply by using three-part names to perform cross-database queries.

**Comment by ArmandoLacerda** [comment]
@bradleyschacht 's version of the paragraph is way better than mine. Most important, it corrects the mistakes from the original paragraph, which is my intention. 

I vote to use the new proposed text by Bradley.

File: docs/get-started/decision-guide-data-store.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ Susan, a professional developer, is new to Microsoft Fabric. They're ready to ge
...
 
-Susan creates a new lakehouse and accesses the data warehouse capabilities with the lakehouse SQL analytics endpoint. Using the Fabric portal, creates shortcuts to the external data tables and places them in the `/Tables` folder. Susan now can write T-SQL queries that reference shortcuts to query Delta Lake data in the lakehouse. The shortcuts automatically appear as tables in the SQL analytics endpoint and can be queried with T-SQL using three-part names.
+Susan creates a new data warehouse and access it as it is a SQL server database. Using the Fabric portal, she creates references to other lakehouses and data warehouses using cross-database query technics. Susan now can write T-SQL queries that reference Delta Tables in the lakehouse. She can query data using T-SQL three-part names.
```
Comment (Court72) [inline docs/get-started/decision-guide-data-store.md:75]:
```suggestion
Susan creates a new data warehouse and interacts with it using T-SQL just like her other SQL server databases. Most of the existing T-SQL code she has written to build her warehouse on SQL Server will work on the Fabric data warehouse making the transition easy. If she chooses to, she can even use the same tools that work with her other databases, like SQL Server Management Studio. Using the SQL editor in the Fabric portal, Susan and other team members write analytic queries that reference other data warehouses and Delta tables in lakehouses simply by using three-part names to perform cross-database queries.
```

**Comment by SnehaGunda** [comment]
#sign-off

**Pull request merged**